static/app/views/detectors/detectorListTable.tsx

```diff
@@ -0,0 +1,193 @@
+
+  const detectorIds = useMemo(() => detectors.map(detector => detector.id), [detectors]);
+
+  const handleSelect = useCallback(
+    (id: string, checked: boolean): void => {
+      if (checked) {
+        setSelectedRows([...selectedRows, id]);
+      } else {
+        setSelectedRows(selectedRows.filter(item => item !== id));
+      }
+    },
+    [selectedRows]
+  );
+
+  function selectAll(): void {
+    if (selectedRows.length === 2) {
+      setSelectedRows([]);
+    } else {
+      setSelectedRows(detectorIds);
+    }
+  }
+
+  const bulkActionsVisible = useMemo(() => selectedRows.length > 0, [selectedRows]);
+  const canDelete = false;
+
+  return (
+    <Panel>
+      <StyledPanelHeader>
+        <Flex>
+          <StyledCheckbox
+            checked={selectedRows.length === detectors.length}
+            onChange={selectAll}
+            visible
+          />
+          <Flex justify={'space-between'} flex={1}>
+            {bulkActionsVisible ? (
+              <ActionsWrapper {...animationProps}>
+                <Button size="xs">{t('Disable')}</Button>
+                {canDelete ? (
+                  <Button size="xs" priority="danger">
+                    {t('Delete')}
+                  </Button>
+                ) : (
+                  <Subtext>{t("Occurrence-based monitors can't be deleted")}</Subtext>
+                )}
+              </ActionsWrapper>
+            ) : (
+              <Heading {...animationProps}>{t('Name')}</Heading>
+            )}
+          </Flex>
+        </Flex>
+        <Flex className="last-issue">
+          <HeaderDivider />
+          <Flex justify="space-between" flex={1}>
+            <Heading>{t('Last Issue')}</Heading>
+            <StyledIconArrow
+              size="xs"
+              direction={direction === 'desc' ? 'down' : 'up'}
+              color="subText"
+              onClick={() => setDirection(direction === 'desc' ? 'asc' : 'desc')}
+            />
+          </Flex>
+        </Flex>
+        <Flex className="open-issues">
+          <HeaderDivider />
+          <Heading>{t('Open Issues')}</Heading>
+        </Flex>
+        <Flex className="connected-automations">
+          <HeaderDivider />
+          <Heading>{t('Connected Automations')}</Heading>
+        </Flex>
+      </StyledPanelHeader>
+      <PanelBody>
+        {detectors.map(detector => (
+          <DetectorListRow
+            key={detector.id}
+            automations={detector.automations}
+            groups={detector.groups}
+            id={detector.id}
+            link={detector.link}
+            name={detector.name}
+            project={detector.project}
+            details={detector.details}
+            handleSelect={handleSelect}
+            selected={selectedRows.includes(detector.id)}
+            disabled={detector.disabled}
+          />
+        ))}
+      </PanelBody>
+    </Panel>
+  );
+}
+
+const HeaderDivider = styled('div')`
```

Comment:
Seems like this could be accomplished with a `border` rather than `<div />`s?

Reply:
I tried this, but I couldn't find a way to round the ends of the dividers that way?